pkg/user/tx_client.go

```diff
@@ -306,27 +305,19 @@ func SetupTxClient(
 
 // SubmitPayForBlob forms a transaction from the provided blobs, signs it, and submits it to the chain.
 // TxOptions may be provided to set the fee and gas limit.
-// This method uses the tx queue infrastructure and blocks until the transaction is confirmed.
+// This method broadcasts the transaction and waits for confirmation using the default account.
 func (client *TxClient) SubmitPayForBlob(ctx context.Context, blobs []*share.Blob, opts ...TxOption) (*TxResponse, error) {
-	if client.txQueue == nil {
-		return nil, errors.New("tx queue not configured")
-	}
-
-	if !client.txQueue.started.Load() {
-		return nil, errors.New("tx queue not started")
-	}
+	return client.SubmitPayForBlobWithAccount(ctx, client.defaultAccount, blobs, opts...)
```

Comment:
I know we already discussed this but i might have missed something. Why did we decide to keep SubmitPFB as is? Since IIUC `SubmitPayForBlobInQueue` with 1 worker should work the same way.

Reply:
> SubmitPayForBlobInQueue with 1 worker should work the same way.

the diff is just that this won't submit more than one blob at a time when called  multiple times as all txs enter the queue. for example if we had 10 transactions  via SubmitPFB, they would be included potentially in a single block. While rare it does occur occasionally. Where as queue guarantees this takes at minimum 10 blocks